mlflow/ml-package-versions.yml

```diff
@@ -190,6 +190,9 @@ gluon:
   models:
     minimum: "1.5.1"
     maximum: "1.9.1"
+    requirements:
+      # numpy type declarations are fully deprecated
+      ">= 1.6.0": ["numpy<1.24.0"]
```

Comment:
Can we add upper bounds in all places where we're pinning numpy?

Reply:
Inverted the two unbounded tests